Release 1.1.0 (#732)
ably/ably-asset-tracking-android: README.md

Visit the [Ably Asset Tracking](https://ably.com/docs/asset-tracking) documentation for a complete API reference and code examples.

#### Upgrade / Migration Guide

> Please see our [Upgrade / Migration Guide](UPDATING.md) for notes on changes you need to make to your code to update it to use new APIs.

Reviewer: ```suggestion
Please see our [Upgrade / Migration Guide](UPDATING.md) for notes on changes you need to make to your code to update it to use the latest version of these SDKs.
```
